### PR TITLE
Adds APC and different areas for the multiple air alarms.. why could you siphon interrogation from perma..

### PR DESCRIPTION
**Pull request by JamieD1**
:cl:  
tweak: Fixes a couple air alarms and missing apc in brig
/:cl:

### CODE REVIEW

**Review by ToasterBiome — CHANGES_REQUESTED** [review]
Bad areastrings on the APCs

**Review by ToasterBiome — CHANGES_REQUESTED** [review]
can you look on our other maps in rotation to see if they could also benefit from these new areas?

File: code/game/area/Space_Station_13_areas.dm [hunk 1]
```diff
@@ -850,6 +850,14 @@ NOTE: there are two lists of areas in the end of this file: centcom and station
 	minimap_color = "#b12527"
 	airlock_wires = /datum/wires/airlock/security
 
+/area/security/main
+	name = "Security Office"
+	icon_state = "security"
+
```
Comment (JamieD1) [inline code/game/area/Space_Station_13_areas.dm]:
```suggestion
```

**Review by ToasterBiome — CHANGES_REQUESTED** [review]
Box's prison hallway APC has the same name as the prison wing APC (for power inside perma)

Meta is missing the area + apc for prison hallway, despite being around the same size as box's

Delta does not have the prison hallway area + apc, despite being bigger than box's

Kilo does not have the prison hallway area + apc, despite being a shit map

**Review by ToasterBiome — APPROVED** [review]
ye looks good

**Pull request merged**